tests/actions/ReportTest.js

```diff
@@ -329,9 +329,9 @@ describe('actions/Report', () => {
                 expect(utcToZonedTime(report.lastReadTime, UTC).getTime()).toBeGreaterThanOrEqual(utcToZonedTime(currentTime, UTC).getTime());
                 expect(report.lastMessageText).toBe('Current User Comment 3');
 
-                const USER_1_BASE_ACTION = {
+                const USER_2_BASE_ACTION = {
```

Comment:
Could you explain these test changes? I don't exactly understand what is happening here.

Reply:
@cubuspl42  After a thorough examination, it seems that there is a test being conducted to mark one's own comment as unread. The crucial question emerges: how does one actually mark their own comment as unread? It's important to highlight that User 1 is signed in during the initial phase of the test. I have just ensured it's the comment of User 2 that is being tested for the ability to be marked as unread. That'all

Reply:
> how does one actually mark their own comment as unread? 

Like this:

![image](https://github.com/Expensify/App/assets/2590174/497a81db-3544-4e9e-a25e-ee924e8f0339)

Reply:
Would it be easy to hide the "Mark as unread" option? We'd also need to conditionally hide this one:

![image](https://github.com/Expensify/App/assets/2590174/31694f47-1cb2-4a15-b27b-76f85eac7975)

Honestly, I didn't really think much about this case during proposal review. It's a corner case, true, but it doesn't mean we should break app's current behavior 🙁

Reply:
> Would it be easy to hide the "Mark as unread" option for one's own actions? We'd also need to conditionally 

Yeah, we can do it. Not a big deal

Plus do you think this makes sense: If the most recent comment or message is from the user who owns the account, then no preceding message can be marked as unread?

If it does I'll add those changes too.

Reply:
> Plus do you think this makes sense: If the most recent comment or message is from the user who owns the account, then no preceding message can be marked as unread?

Oh, there's also that case... It's getting complicated...

If we go this road, the next step is disallowing to mark any action before "our" action as unread. And I wouldn't like to go there.

I didn't have strong feelings about hiding the "Mark as unread" option. I just want to keep things consistent. And with the code we have so far, things are not consistent in my opinion.

I'd like to keep this invariant:

If you execute the "Mark as unread" operation, something should be marked as unread. A green line should appear in the chat, bold text in LHN, and all that.

So either it's always possible to run "Mark as unread", and these things happen, or it's sometimes not possible to run "Mark as unread".

But the latter is just a solution to a problem with a solution to a problem.

Reply:
Also, we'd need to get approval for changing the "Mark as unread" behavior. My approval is not enough. The bigger the change is, the more difficult it's gonna be.

Reply:
Write this clearly
Looks like issue: https://github.com/Expensify/App/issues/35575 
is no longer a problem. I checked in both the staging and main. Now we can't reproduce this problem anymore. Force offline option from preference is also removed. Even if we off our network and repeat the steps in the issue. LHN shows read.

Can you verify it? Looks like we are dealing with the problem already solved.